test/runtests.jl

```diff
@@ -184,6 +196,22 @@ seek(io, 0)
 s = read(io, String)
 @test s[1:17] == "ParametricSpline("
 
+
+# test equality
+seed!(0)
+x = sort(rand(10))
+y = rand(3, 10)
+sp1 = ParametricSpline(x, y)
+sp2 = ParametricSpline(x, y)
+sp3 = ParametricSpline(x.+1, y)
+sp4 = ParametricSpline(x, y.+1)
+@test sp1 == sp2
+@test allunique([sp1,sp3,sp4])
+
+
+println("All tests passed.")
```

Comment:
Minor: seems weird to have this print out not at the end. Maybe move to the end?

Reply:
Yes, looks like I accidentally moved it when copy-pasting stuff. Fixed.